gradle.properties

```diff
@@ -50,12 +52,13 @@ spotbugs.version=4.2.1
 #
 # plugins
 #
-plugin.shadow.version=4.0.4
+plugin.shadow.version=6.1.0
 plugin.license.version=0.15.0
-plugin.dependencycheck.version=6.1.2
+plugin.dependencycheck.version=6.1.5
 plugin.spotbugs.version=4.7.0
 plugin.forbiddenapis.version=3.1
 plugin.utf8.version=0.1.0
 plugin.metadata.version=0.2.0
 plugin.javadoc-links.version=0.3.0
 plugin.nexus-publish.version=1.0.0
+plugin.versions.version=0.38.0
```

Comment:
the naming confuses me a bit tbh

Reply:
I tried to stay consistent with our other projects